### ISSUE TITLE
A Light Autoencoder Networks for Face Swapping

### ISSUE THREAD

**Issue by npresseller** [npresseller/Capstone-Bibliography#13]
### Title
A Light Autoencoder Networks for Face Swapping
### URL
https://dl.acm.org/doi/10.1145/3297156.3297210
### Summary 
Gives background on generative modeling and autoencoding techniques
### Key Points 
Deep modeling and Autoencoder-
     Autoencoder can be understood as a generation model which
decodes the input through restoration by the hidden layer
information. Therefore, after the concept of the encoder and the
decoder is broadened, the encoding phase and the decoding phase
can be understood as random mappings.
The reason why the autoencoder is similar to the neural network is
that the autoencoder can be regarded as a special case of the
feedforward network. When training, the same technique is used
to perform forward and backward operations with the neural
network. And the loss function is set, then, using the small batch
gradient descent method, the weight values in the autoencoder is
updated. As shown in fig.2, the autoencoder takes the input
images through the network to obtain implicit information, and
the decoder is responsible for decoding and producing the implicit
information to restore the input images. 

Face-swapping-
Face swapping is a widely used technique in the field of computer
vision. Generally, it can be used for video synthesis, privacy
services, portrait replacement and other innovative applications.
Typical face swapping was achieved by analyzing the similarity
information of the faces of the two faces separately which does
not require training and dataset [1]. However, the performance of
the traditional method is relatively poor and it cannot modify the
expression in the face.
Fortunately, with the development of artificial intelligence, one
key advantage of the deep learning technique is that the deep
information of the input image can be extracted through the deep
neural network. Moreover, the hidden deep features can also be
obtained to achieve some novel tasks, such as Style Transfer [2,3]
and Generative Adversarial Networks (GANs) [4,5]. Both of these
can reproduce wonderful images from deep network. Especially
GAN, whose generator [6] is an encoder-decoder network which
tries to synthesize fake image conditioned on a given image to
fool discriminator, while the discriminator tries to identify the
fake image by comparing with the corresponding target image.
However, this work is hard to train, only through rough
adjustment can get relatively good results. 
### Citation
@inproceedings{10.1145/3297156.3297210,
author = {Guo, Yanzong and He, Wangpeng and Zhu, Juanjuan and Li, Cheng},
title = {A Light Autoencoder Networks for Face Swapping},
year = {2018},
isbn = {9781450366069},
publisher = {Association for Computing Machinery},
address = {New York, NY, USA},
url = {https://doi.org/10.1145/3297156.3297210},
doi = {10.1145/3297156.3297210},
abstract = {Face swapping in images is a process where an input identity is transformed into a target identity while preserving the former's facial expression, pose and lighting. In this paper, a new face swap method using autoencoder networks is investigated, which can capture the two different image sets to be trained. The images from two sets are unstructured collection of people's photographs. This approach is enabled by framing the face swapping problem in terms of style transfer, where the goal is to render an image in the style of another one. We design a new autoencoder network that enables the processing to produce photorealistic results. Moreover, in low resolution video, the proposed method can achieve face swap by combining autoencoder network with pre- and post-processing steps.},
booktitle = {Proceedings of the 2018 2nd International Conference on Computer Science and Artificial Intelligence},
pages = {459–462},
numpages = {4},
keywords = {computer vision, machine learning, autoencoder, deep learning},
location = {Shenzhen, China},
series = {CSAI '18}
}
